File: 9-PalindromeNumber/9-PalindromeNumber.java
// Last updated: 11/11/2025, 1:58:18 AM
class Solution {
    public boolean isPalindrome(int x) {
        String str = Integer.toString(x);
        String tempStr = "";
        for (int j = (str.length()-1); j>=0; j--) {
            tempStr = tempStr + str.charAt(j);
        }
        return (tempStr.equals(str));
    }
}

File: 9-PalindromeNumber/9-PalindromeNumber.py
# Last updated: 11/11/2025, 1:58:16 AM
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
    
        string = str(x)
        return (string == string[::-1])
        